cmd/helm/fetch.go

```diff
@@ -138,6 +147,43 @@ func downloadChart(pname string, untar bool, untardir string, verify bool, keyri
 	return saveChart(pname, buf, untar, untardir)
 }
 
+// findChartUrl finds the download URL for a chart in an index
+//
+// Lets assume pnane is of the form 'repo/chartname-version' or 'repo/chartname-version.tgz'
+// if there is no 'repo/' prefix we will search all repositories
+func findChartURL(pnane string) (*url.URL, error) {
```

Comment:
hey @jstrachan - I'm a little confused here. Would you mind clarifying what the objective of this function is? I think the objective is not to have to pass in the repo prefix, but that is important when identifying which chart you are actually trying to fetch because two different repos can have charts with the exact same name and version.